requirements.txt

```diff
@@ -10,3 +10,4 @@ botocore==1.20.95
 deprecation==2.1.0
 urllib3==1.26.18
 tqdm==4.62.3
+pint==0.18
```

Comment:
Since the constraint on pint wasn't present in gemd 1.16.0, a clean install (such as our tests do) results in 0.23 being downloaded, and thus errors. And when I tried pinning it at 0.20, I also got errors, I think because they dropped 3.7 support.

So this will go away once I do https://citrine.atlassian.net/browse/PLA-13742, which I will be starting on today.